src/main/webapp/app/app.tsx

```diff
@@ -72,6 +73,20 @@ export class App extends React.Component<IAppProps> {
           </div>
           <Footer />
         </div>
+      </div>
+    );
+
+    return (
+      <Router>
+        <Switch>
+          <PrivateRoute
```

Comment:
the measure-layer utility might break for the new route if we won't have measure-layer div there